crates/router/src/connector/utils.rs

```diff
@@ -199,7 +206,7 @@ impl PaymentsAuthorizeRequestData for types::PaymentsAuthorizeData {
                 .is_some()
     }
     fn get_webhook_url(&self) -> Result<String, Error> {
-        self.router_return_url
```

Comment:
did you checkout whether this is being used anywhere? if it is used, change that to `get_return_url`

Reply:
Added `get_router_return_url`